themes/build.js

```diff
@@ -0,0 +1,96 @@
+const request = require("request");
+const less = require("less");
+const fs = require("fs");
+const { variants, labels } = require("@catppuccin/palette");
+
+const themes = ["latte", "frappe", "macchiato", "mocha"];
+
+const url = "https://raw.githubusercontent.com/catppuccin/palette/main/less/";
+
+fs.mkdirSync("built");
+
+console.log("Reading theme.less file");
+fs.readFile("src/theme.less", "utf8", (err, data) => {
+	if (err) {
+		console.error(err);
+		return;
+	}
+	// iterate through themes
+	themes.forEach((theme) => {
+		// replace appearance with .light or .dark
+		const themeData = data.replace(
+			/"appearance"/g,
+			theme === "latte" ? ".light" : ".dark"
+		);
+
+		// fetch the less file from github
+		console.log("Fetching less file from GitHub");
+		request(url + theme + ".less", (err, res, body) => {
+			if (err) {
+				console.error(err);
+				return;
+			}
+
+			for (let colorName in variants[theme]) {
```

Comment:
Prefer `for (const value of iterable)`

Reply:
I have updated it to.
```js
for (const colorName in variants[theme]) {
```
Using `of` seems to fail. Would I be missing something? Variants[theme] returns unable to iterate when using the `of` keyword.